### ISSUE TITLE
[improvement] Load dataset through url parameters

### ISSUE THREAD

**Issue by qchenevier** [vega/voyager#893]
Hello,

I've been loving this app for years, very nice work. It's very useful to explain how to do exploratory data analysis to students, without the need to open a notebook.

Recently, I've been willing to use the app for a data viz course I'll do in coming weeks and thought that it would be great to be able to load a dataset thanks to an url parameter. Something like:
```https://vega.github.io/vega-lite/data/cars.json?dataset=https://vega.github.io/vega-lite/data/cars.json```

It would load the dataset without the need to open the "load dataset pane".

In this example, this dataset is obviously already available in the app... but for my course, I'm using a dataset which is not part of the vega datasets and I need to explain to students how to load the dataset before starting the session. Not a huge pain point, but still I think it would be great if they could open the app with the right dataset simply by clicking a link.

I guess it could be done using by modifying [the DataSelector component](https://github.com/vega/voyager/blob/master/src/components/data-selector/index.tsx).

If you like the idea, I may propose a PR someday, but since I'm not fluent in React ... it may be long.

What do you think ?

**Comment by domoritz** [comment]
Hi @qchenevier. Thanks for filing the issue. It's great to hear you are enjoying voyager and find it useful. 

I like your idea and think it would be a great addition. We already have support for sharing specs in the Vega Editor so having something similar for data in Voyager makes a lot of sense. I won't have cycles to implement it but I am supportive of adding the feature.
